### ISSUE TITLE
[FEATURE REQUEST] User Interface for Android Auto

### ISSUE THREAD

**Issue by Fakorede** [namidaco/namida#503]
**Is your feature request related to a problem? Please describe.**
Currently, the application lacks user interface support for Android Auto, limiting its functionality and accessibility for users while driving.

**Describe the solution you'd like**
- Application should have a dedicated interface for displaying core media features through the Android Auto UI as recommended [here](https://developer.android.com/training/cars/media#implement_browser)

**Additional context**

<img width="795" alt="Image" src="https://github.com/user-attachments/assets/2ff10f29-790b-4907-8575-5a07054440e7" />

**Apps**

[Refreezer](https://github.com/DJDoubleD/refreezer/blob/cd046cdfab9fb39afe4c5b4c1223706156f76fe1/lib/service/audio_service.dart#L29) is another Flutter based app that does this really well.

**Comment by MSOB7YY** [comment]
hey this is pretty much #157 which requests android auto support, im still not sure about when to do it but thanks for the example app and for the docs reference!

will close this as duplicate :)

**Comment by MSOB7YY** [comment]
hey i did some changes in v5.0.1, can u check it out and see if android auto works propely? https://github.com/namidaco/namida-snapshots/releases

**Comment by yourassalt** [comment]
Thanks man

**Comment by MSOB7YY** [comment]
@yourassalt thanks what? does it work or not? 😂

**Comment by yourassalt** [comment]
It works man you got me defensive with this now I have to think 10 times
before I reply 🤣🤣

**Comment by MSOB7YY** [comment]
hgahgahga im glad for that and nah dw i was just making sure cuz i couldnt believe it 💀😂

**Comment by yourassalt** [comment]
Na it's fine 👌🤣 but it works now perfectly 🎉🎉

**Comment by Fakorede** [comment]
Great job! Can confirm there's a ui now
